### PR TITLE
【BIT】转换能力补齐：paddle.nn.functional.lp_pool[1 2]d

### PR DESCRIPTION
**Pull request by cangtianhuang**
### 1. [paddle.nn.functional.lp_pool1d](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/lp_pool1d_cn.html#lp-pool1d)

查询 [torch.nn.functional.lp_pool1d 与 paddle.nn.functional.lp_pool1d 对照表](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.lp_pool1d.html) ，发现是 `paddle 参数更多`，但是所有参数的转写逻辑与 paddle.nn.functional.avg_pool[1 2 3]d 类似，因此编写 mapping 转换配置后，直接复用 PoolRule 类

将 PoolRule 类移动至 p 位置

### 2. [paddle.nn.functional.lp_pool2d](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/lp_pool2d_cn.html#lp-pool2d)

查询 [torch.nn.functional.lp_pool2d 与 paddle.nn.functional.lp_pool2d 对照表](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.lp_pool2d.html) ，发现是 `paddle 参数更多`，编写 mapping 转换配置后，复用 PoolRule 类

## 测试结果

paddle.nn.functional.lp_pool[1 2]d 的所有配置均通过 accuracy 测试，移入 support2torch_1

image

### CODE REVIEW

**Review by yuwu46 — APPROVED** [review]

**Review by wanghuancoder — APPROVED** [review]
LGTM
